util/config.go

```diff
@@ -333,6 +384,10 @@ func ParseConfigFromJson(content string) *Config {
 }
 
 func ParseConfigFromFile(filePath string) *Config {
+	kavaConfig := sdk.GetConfig()
+	app.SetBech32AddressPrefixes(kavaConfig)
```

Comment:
can we move this out

Reply:
https://github.com/binance-chain/bep3-deputy/pull/1/commits/bf11706ef79fb3546a57768534b75c309ca7697e